### PR TITLE
Revert the default policy server finalizer in use

### PR DESCRIPTION
**Pull request by jvanz**
## Description

This reverts commit e514fa7c65c45f3d631544e15c6a55208169a765. This is necessary because the controller using the same finalizer is not released yet.

### CODE REVIEW

**Review by kravciak — APPROVED** [review]

**Comment by viccuad** [comment]
Note , this is a duplicate of https://github.com/kubewarden/helm-charts/pull/467 xD. I'm ok with this one, closing the other.

**Review by viccuad — APPROVED** [review]

File: charts/kubewarden-defaults/Chart.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ keywords:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.0.2
+version: 2.0.3
```
Comment (viccuad) [inline charts/kubewarden-defaults/Chart.yaml:25]: we need a bump in `catalog.cattle.io/upstream-version, too.

**Review by viccuad — CHANGES_REQUESTED** [review]

**Comment by jvanz** [comment]
> Note , this is a duplicate of #467 xD. I'm ok with this one, closing the other.

Sorry, I've missed your PR. Closing this one.

**Comment by jvanz** [comment]
@viccuad I've tried to update  your PR (using force push) to change a commit message and did a mess in the branch. Losing the commit with the fix. I'm sorry. I've fixed your comment in this PR and I'll merge it.

**Comment by viccuad** [comment]
@jvanz no problem! I'm glad that is out!

**Pull request merged**